src/OpenDebugAD7/AD7Resources.resx

```diff
@@ -304,4 +304,12 @@
   <data name="Error_UnableToSetInstructionBreakpoint" xml:space="preserve">
     <value>Error setting instruction breakpoint: {0}</value>
   </data>
+  <data name="Error_FilterOption_Not_Supported" xml:space="preserve">
+    <value>'{0}' is not a supported exception filter.</value>
+    <comment>{0} is the name of the exception filter.</comment>
+  </data>
+  <data name="Error_Invalid_Exception_Condition" xml:space="preserve">
+    <value>ERROR: '{0}' is an invalid exception condition</value>
```

Comment:
>condition [](http://example.com/codeflow?start=48&length=9)

Include aka.ms link to documentation?

Reply:
Also need to update https://github.com/microsoft/vscode-cpptools/blob/main/Documentation/Debugger/ExceptionSettings.md for the cppdbg case.